code/datums/mood_events/generic_positive_events.dm

```diff
@@ -198,3 +198,8 @@
 /datum/mood_event/confident_mane
 	description = "<span class='nicegreen'>I'm feeling confident with a head full of hair.</span>\n"
 	mood_change = 2
+
+/datum/mood_event/holy_consumption
+    description = "<span class='nicegreen'>Truly that was the food of the Divine!.</span>\n"
```

Comment:
```suggestion
    description = "<span class='nicegreen'>Truly, that was the food of the Divine!.</span>\n"
```

Reply:
you did a spell check comment and didnt notice the `!.`

Reply:
whoopsie doopsie